### PR TITLE
Add responsibility of maintainers from ChakraCore

### LINKED ISSUE #83: add responsibility of maintainers section a la chakracore
add this to our CoC cuz its rad from [Microsoft/ChakraCore]:

> Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.

> By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.

[Microsoft/ChakraCore]: https://github.com/Microsoft/ChakraCore/blob/master/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Charlotteis**
Closes #83 

Thanks for the opportunity to contribute!

### CODE REVIEW

**Comment by beaugunderson** [comment]
:+1: from me :)

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -32,6 +32,17 @@ The definitions of various subjective terms such as "discriminatory",
 "hateful", or "confusing" will be decided at the sole discretion of
 the Node.js Inclusivity Working Group team members.
 
+Project maintainers have the right and responsibility to remove, edit,
```
Comment (nebrius) [inline CODE_OF_CONDUCT.md]: Can we change `Project Maintainers` here and below to `Working Group members`? To me, `Project Maintainers` references maintainers of the Node.js project, which is a lot more broad than this working group.

Comment (ashleygwilliams) [inline CODE_OF_CONDUCT.md]: A++, once this is changed i am happy to merge!

**Comment by Charlotteis** [comment]
Changed each mention to 'Working Group members', include amending 'removal from the project team' to 'removal from the Working Group'. Hope that's better now.

**Comment by nebrius** [comment]
Love it! LGTM :)

**Pull request merged**